src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String SPECIFIED_CHARACTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        String[] resultArray;
+        ArrayList<String> result = new ArrayList<>();
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
+            String stringOfFile = bufferedReader.readLine();
+            while (stringOfFile != null) {
+                stringBuilder.append(stringOfFile).append(" ");
```

Comment:
you don't need this stringBuilder. Split the line by "\\\W+" regex and add filtered words to your list here, that's will make the code shorter

Reply:
done